pkg/virt-controller/watch/migration.go

```diff
@@ -39,8 +39,6 @@ import (
 	"k8s.io/apimachinery/pkg/api/equality"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Is this necessary? Why not to use `metav1` from the line above?

Reply:
I removed the duplicated import package `v1 "k8s.io/apimachinery/pkg/apis/meta/v1"`, using `metav1 ` above it.

Reply:
Oh! Sorry for my confusion. Makes total sense, thank you.

Reply:
Still, please do the removal of duplicated import in a separate commit so it gets easier to see the actual proposed fix (instead of multiple things happening together)

Reply:
> Still, please do the removal of duplicated import in a separate commit so it gets easier to see the actual proposed fix (instead of multiple things happening together)

OK, I will fix it in another commit.